### PR TITLE
Make blockpc exploitation optional

### PR DESCRIPTION
**Pull request by rsas**

### CODE REVIEW

**Comment by regehr** [comment]
Please add sclang support.

**Comment by rsas** [comment]
Done.

File: lib/Extractor/Candidates.cpp [hunk 1]
```diff
@@ -35,6 +36,11 @@
 #include <unordered_set>
 #include <tuple>
 
+static llvm::cl::opt<bool> ExploitBPCs(
+    "souper-exploit-bpcs",
```
Comment (chenyang78) [inline lib/Extractor/Candidates.cpp]: How about souper-exploit-blockpcs? I think it doesn't hurt to use a complete name here.

Comment (rsas) [inline lib/Extractor/Candidates.cpp]: Agrred.

**Comment by rsas** [comment]
Is it good to squash?

**Comment by chenyang78** [comment]
LGTM. Thanks.

**Pull request merged**